src/main/java/apoc/load/util/LoadJdbcConfig.java

```diff
@@ -0,0 +1,50 @@
+package apoc.load.util;
+
+import java.time.ZoneId;
+import java.util.Collections;
+import java.util.Map;
+
+import static apoc.util.Util.toBoolean;
+
+/**
+ * @author ab-Larus
+ * @since 03-10-18
+ */
+public class LoadJdbcConfig {
+
+    private ZoneId zoneId;
+    private boolean localDateTime;
+    private boolean localDate;
+    private boolean zoneDateTime;
+    private boolean offSetTime;
+
+    public LoadJdbcConfig(Map<String,Object> config) {
+        config = config != null ? config : Collections.emptyMap();
+        this.zoneId = (ZoneId) config.getOrDefault("timezone",ZoneId.of("UTC"));
```

Comment:
what if more than one of them is true? Should we validate that?

I think a different format is better: `{temporal: "localdatetime", timezone: 'UTC'}`

with the names from the cypher temporal functions